app/services/organizations/profile_image_generator.rb

```diff
@@ -0,0 +1,7 @@
+module Organizations
```

Comment:
Since this is basically a copy of the User version, would it be maybe better to refactor that object to not be `Users::` so that we re-use a single object here?

Reply:
Sure thing! moved this and the safe remote profile image url service to the Images namespace